README.md

```diff
@@ -314,6 +315,33 @@ ReactDOM.render(<Clock />, document.getElementById('react'))
 
 Try this live: <https://jsfiddle.net/arunoda/wm6romh4/>
 
+## Extending
```

Comment:
But, I like to do this other way around. Add these lifecycle hooks for the `Time` component.

Is there any advantage on this approach?